adcc/modified_transition_moments.py

```diff
@@ -72,10 +74,43 @@ def compute_modified_transition_moments(gs_or_matrix, dipole_operator,
                         "libadcc.OneParticleOperator.")
 
     if method.is_core_valence_separated:
-        raise NotImplementedError("compute_modified_transition_moments "
-                                  "not implemented for CVS")
-
+        if method.name == "cvs-adc1":
+            return AmplitudeVector(dipole_operator['o2v1'])
+        else:
+            cvs_mtm = Adc2CVSModifiedTransitionMoments(method,
+                                                       ground_state,
+                                                       dipole_operator)
+            return cvs_mtm.cvs_adc2_mtm()
     mtm_cpp = libadcc.compute_modified_transition_moments(
         method.name, ground_state, dipole_operator
     )
     return AmplitudeVector(*mtm_cpp.to_tuple())
+
+
+class Adc2CVSModifiedTransitionMoments:
+    def __init__(self, method, ground_state, dipole_operator):
+        self.ground_state = ground_state
+        self.cvs_matrix = AdcMatrix(method, ground_state)
+        self.intermediates = self.cvs_matrix.intermediates
```

Comment:
If `gs_or_matrix` in the `compute_modified_transition_moments` is a matrix it should be used here in order to re-use the intermediates.